frontend/src/scenes/authentication/signup/signupForm/test/Signup.stories.tsx

```diff
@@ -0,0 +1,69 @@
+// Signup.stories.tsx
+import { Meta } from '@storybook/react'
+import { useEffect } from 'react'
+import { SignupForm } from './SignupForm'
+import { mswDecorator, useStorybookMocks } from '~/mocks/browser'
+import { userLogic } from 'scenes/userLogic'
+import preflightJson from '~/mocks/fixtures/_preflight.json'
+
+export default {
+    title: 'Scenes-Other/Signup',
```

Comment:
If you want to have test version stories, they need to be named differently from the control ones otherwise Storybook will complain as it will read these as duplicates

Reply:
Good call. Moved them out of the test/control directories.